rslib/src/import_export/text/import.rs

```diff
@@ -625,10 +632,8 @@ impl ForeignCard {
         }
     }
 
-    fn native_due(self, today: u32) -> i32 {
-        let remaining_secs = self.interval as i64 - TimestampSecs::now().0;
-        let remaining_days = remaining_secs / (60 * 60 * 24);
-        0.max(remaining_days as i32 + today as i32)
+    fn native_due(self, timing: &SchedTimingToday) -> i32 {
+        (self.due - timing.next_day_at.0 as i32) / 86_400
```

Comment:
Don't we still need to adjust by `timing.days_elapsed`? And wouldn't it be more correct to round up?
```suggestion
        (self.due - timing.next_day_at.0 as i32 + 86_400 - 1) / 86_400 + timing.days_elapsed as i32
```
(Code not checked.)

Reply:
:man_facepalming: thanks Rumo, silly oversights on my part.